record-and-playback/core/lib/recordandplayback/edl/video.rb

```
@@ -312,6 +315,55 @@ def self.render(edl, layout, output_basename, threads = 2)
+          end
+        end
+
+        if videos_to_resample_due_to_gaps.any?
+          BigBlueButton.logger.info "Resampling #{videos_to_resample_due_to_gaps.length} video(s) with large PTS gaps (segmented approach)."
+          resampled_videos_dir = File.join(File.dirname(output_basename), "resampled_videos_segmented")
+          FileUtils.mkdir_p(resampled_videos_dir)
+          
+          filename_update_map_gaps = resample_video_files(videos_to_resample_due_to_gaps, resampled_videos_dir, videoinfo)
+
+          if filename_update_map_gaps.any?
+            BigBlueButton.logger.info "Updating EDL and video information for segment-resampled videos."
```

RuboCop (1.82.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)